packages/tupaia-web/src/features/Map/MapOverlaySelector/MapTableModal.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import DownloadIcon from '@material-ui/icons/GetApp';
+import MuiIconButton from '@material-ui/core/IconButton';
+import { FlexColumn } from '@tupaia/ui-components';
+import { Modal } from '../../../components';
+import { MapTable, MeasureData } from '@tupaia/ui-map-components';
+import { useParams } from 'react-router';
+import { useEntityAncestors, useMapOverlays, useEntitiesWithLocation } from '../../../api/queries';
+import { useMapOverlayReport } from '../utils';
+import { processMeasureData } from '../MapOverlays/processMeasureData';
+
+const Wrapper = styled(FlexColumn)`
+  justify-content: flex-start;
+`;
+
+const Title = styled.div.attrs({
```

Comment:
In this case, because this is an `h2`, we should use the `Typography` component instead of a `div`. Still keep all the rest, but we just want to avoid using divs anywhere that has a better semantic alternative.